Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -51,8 +91,30 @@ public static int[] RemoveFirst(int[] array)
         /// <returns>A new array with element removed at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveAt(int[] array, int index)
         {
-            // ToDo: implement.
-            return array;
+            if (array == null)
```

Comment:
Duplicate code with removal in other places.
Ideally, reuse this function when removing first or last by passing a proper index.